### ISSUE TITLE
question

### ISSUE THREAD

**Issue by mezvrishvili** [w3lh/w3l#2]
Hello w3lh, i have a question please.

can i change real exeinfo with fake when patching with your loader?

for example 1.26a  has "war3.exe 03/18/11 20:03:55 471040"

need to change 471040 with custom numbers

THanks

**Comment by w3lh** [comment]
war.exe is a file name. 471040 is the size in bytes. Date is written in PE header. Or there is any other place this info came from?

**Comment by mezvrishvili** [comment]
yes, 471040 is file size, but i'm not sure about date.

as its written in versioncheck.conf, its file's date/time. i try to find it out
# exeinfo = "war3.exe ??/??/?? ??:??:?? 1568211" will match war3.exe with    #
#           filesize 1568211 but won't care about the file's date/time 


i'll tell you why i need to change it.

i've compiled new loader, changed name of Game.dll with Antihack.dll, so now Antihack.dll is patched and used by game and maphack not works(all the maphacks use Game.dll)

but i need make so, that players were not able to enter the server with old loader,

so i gonna change pvpgn version check. allow players enter server only if war3.exe has size 111111 for example(or any custom numbers)

Thats why i wanna patch it and provide fake info, as though  war3.exe has size 111111

or maybe patch game.dll and change version manually? as though its not  1.26.0.1 but 1.26.1.2 or something?

**Comment by cen1** [comment]
You are just playing hide and seek game.. if any decent hacker wants to bypass your custom version he can easily do so. It can even be faked on a network level with gproxy for example. I think you are wasting your time to be honest. You can't really verify that a client is genuine in any way that is not obfuscation.

**Comment by mezvrishvili** [comment]
yep, decent hacker can bypass it, but on my server mostly play stupid kids , that are only able to download  maphack and run it :) 

so any simple fix will be ok for me. 

can you please provide more info about ExtraWork?

**Comment by cen1** [comment]
https://github.com/pvpgn/pvpgn-server/issues/72

**Comment by leokinging** [comment]
能否告訴我基於1.29找不到偏移地址嗎？
有1.29調用JASS的基址或者偏移嗎？
資料夾沒有Game.doll只有Warcraft iii

Can you tell me that the offset address cannot be found based on 1.29?
Is there a base address or offset for calling JASS in 1.29?
There is no Game.doll in the folder, only Warcraft iii
